src/ostorlab/runtimes/local/models/models.py

```diff
@@ -657,20 +657,20 @@ def create_from_assets_definition(
         """Create the assets from the asset definition.
 
         Args:
-            scan: The scan object.
             assets: The list of assets to create.
+            scan_id: The scan id.
         """
-        networks: List[str] = []
-        links: List[str] = []
+        networks: List[Dict[str, Union[str, int]]] = []
```

Comment:
the `int` here is for the mask? the proto & asset instance both define it as a string..

Reply:
true, but it doesnt create any conflict here !!